google-cloud-workflows/pom.xml

```
@@ -37,6 +37,10 @@
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
```

```suggestion
      <artifactId>protobuf-java-util</artifactId>
      <scope>test</scope>
```

This artifact is only required for testing and to limit dependencies, we should mark it test scope.

---

Done